File: unix/boot/spp/rpp/ratlibf/prompt.f
      subroutine prompt (str, buf, fd)
      integer str(100), buf(100)
      integer fd
      logical isatty
      if (.not.(isatty(fd)))goto 23000
      call putlin (str, fd)
      call rfflus(fd)
23000 continue
      call getlin (buf, fd)
      return
      end

File: unix/boot/spp/rpp/ratlibr/prompt.r
include	defs

# prompt - write to/read from teletype

   subroutine prompt (str, buf, fd)
   character str(ARB), buf(ARB)
   filedes fd

   logical isatty

   if (isatty(fd))
	 {
	 call putlin (str, fd)
	 call flush (fd)
	 }
   call getlin (buf, fd)

   return
   end
